Utility/ResetToolbars.svg

```diff
@@ -0,0 +1,512 @@
+<?xml version="1.0" encoding="UTF-8" standalone="no"?>
+<!-- Created with Inkscape (http://www.inkscape.org/) -->
```

Comment:
Do you mind exporting as `Plain SVG` file instead of an `Inkscape SVG` file ? (removes inkscape markup from the SVG)